providers/edge/src/airflow/providers/edge/plugins/edge_executor_plugin.py

```diff
@@ -146,9 +147,31 @@ def remove_worker(self, worker_name: str):
     @expose("/status/maintenance/<string:worker_name>/change_comment", methods=["POST"])
     @has_access_view(AccessView.JOBS)
     def change_maintenance_comment(self, worker_name: str):
+        from flask_login import current_user
+
         from airflow.providers.edge.models.edge_worker import change_maintenance_comment
 
         maintenance_comment = request.form.get("maintenance_comment")
+        if maintenance_comment:
```

Comment:
Can you please extract the code of re-writing the comment into a separate function (can be in the same module) and add a pytest for it? Otherwise this is un-tested code.